### PR TITLE
📖Update `gulp visual-diff` instructions in TESTING.md

### PR DESCRIPTION
**Pull request by caroqliu**
cc @ampproject/wg-infra

### CODE REVIEW

File: contributing/TESTING.md [hunk 1]
```diff
@@ -268,16 +268,22 @@ If a Percy test flakes and you would like to trigger a rerun, you can't do that
...
+
+Once the environment variable is set up, you can run the AMP visual diff tests.
 
 First, build the AMP runtime and run the gulp task that invokes the visual diff script:
 
 ```sh
-gulp build
+gulp dist --fortesting
 gulp visual-diff --nobuild
 ```
```
Comment (rsimha) [inline contributing/TESTING.md]: I think this should be split into separate commands. Lines 279-284 can be re-written as:

````
First, build the AMP runtime:

```sh
gulp dist --fortesting
```

Next, run the `gulp` task that invokes the visual diff tests:

```sh
gulp visual-diff --nobuild
```
````

Comment (rsimha) [reply contributing/TESTING.md]: Or if you like, a single block with inline comments would work too:

```sh
# Build the AMP runtime
gulp dist --fortesting

# Run visual tests
gulp visual-diff --nobuild
```

Comment (caroqliu) [reply contributing/TESTING.md]: All set, thanks for the diligence on clarity here!

**Comment by rsimha** [comment]
@danielrozenberg Do you know if something is up with empty Percy builds? 

The [task](https://travis-ci.org/ampproject/amphtml/jobs/614714866#L371) appears to have run fine, but the [build](https://percy.io/ampproject/amphtml/builds/3197684) appears to have stalled.

**Pull request merged**